frontend/packages/console-app/src/components/vpa/VerticalPodAutoscalerRecommendations.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { K8sResourceCommon } from '@console/internal/module/k8s';
+
+export const FilterVerticalPodAutoscaler = (vpas, obj) =>
+  (vpas ?? []).find((vpa) => {
+    const { targetRef } = vpa.spec;
+    return (
+      targetRef &&
+      targetRef.apiVersion === obj?.apiVersion &&
+      targetRef.kind === obj?.kind &&
+      targetRef.name === obj?.metadata?.name
+    );
+  });
+
+export const VerticalPodAutoscalerRecommendations: React.FC<VerticalPodAutoscalerRecommendationsProps> = ({
+  obj,
+}) => {
+  const { t } = useTranslation();
+  const [vpas] = useK8sWatchResource<K8sResourceCommon[]>({
+    groupVersionKind: {
+      group: 'autoscaling.k8s.io',
+      version: 'v1',
+      kind: 'VerticalPodAutoscaler',
+    },
+    namespace: obj?.metadata?.namespace,
+    isList: true,
+    namespaced: true,
+  });
+
+  const verticalPodAutoscaler = FilterVerticalPodAutoscaler(vpas, obj);
+  const targetCPU =
+    verticalPodAutoscaler?.status?.recommendation?.containerRecommendations?.[0]?.target?.cpu;
+  const targetMemory =
+    verticalPodAutoscaler?.status?.recommendation?.containerRecommendations?.[0]?.target?.memory;
+
+  return (
+    <>
+      <dt>{t('console-app~VerticalPodAutoscaler')}</dt>
+      <dd>
+        {targetCPU || targetMemory ? (
+          <>
+            <p>Recommended</p>
+            {targetCPU && (
+              <>
```

Comment:
Nit here and line 53:  you don't need the fragment (`<>`) since you're including a `<div>`.